charts/delphai-job/templates/scaled-job.yml

```diff
@@ -34,7 +34,7 @@ spec:
               - name: AZURE_STORAGE_CONNECTION_STRING
                 valueFrom:
                   secretKeyRef:
-                    name: azure-storage
+                    name: azure-storage-production
```

Comment:
Did we have this secret before you was it created for this change?

Reply:
It was created for this change because it needs to use a storage that isnt normally used in that cluster anymore